src/pages/NotFound/NotFound.tsx

```diff
@@ -0,0 +1,29 @@
+import { ButtonSize, ROUTE, VARIANT } from '@common';
+import { Background, Button } from '@components';
+import { RootLayout } from '@layouts';
+import { Link } from 'react-router-dom';
+
+export const NotFound = () => {
+  return (
+    <RootLayout>
+      <Background />
+      <main className="relative z-2 w-full max-w-7xl mx-auto px-5 flex flex-col justify-center items-center h-[87vh] text-cWhite font-dmsans gap-16">
+        <article className="space-y-lg text-shadow-md">
+          <h1 className="text-7xl font-semibold">Error 404</h1>
+          <h3 className="text-4xl font-semibold">PÁGINA NO ENCONTRADA</h3>
```

Comment:
```suggestion
          <h2 className="text-4xl font-semibold">PÁGINA NO ENCONTRADA</h2>
```

Seguimos los `heading`, de `h1` luego va `h2`